docs/HowToGuides/GettingStarted.md

```diff
@@ -300,6 +302,21 @@ In the following sections, for simplicity, we will assume that you are using a
 unless explicitly mentioned otherwise. You will need to slightly tweak the paths
 for other build configurations.
 
+#### Using both Ninja and Xcode
+
+Some contributors find it more convenient to use both Ninja and Xcode.
+Typically this configuration consists of:
+
+1. A Ninja build created with `--release-debuginfo`.
+2. An Xcode build created with `--release-debuginfo --debug-swift`.
+
+The Ninja build can be used for fast incremental compilation and running tests
+quickly. The Xcode build can be used for debugging with high fidelity.
+
+The additional flexibility comes with two issues: (1) consuming much more disk
+space and (2) you need to maintain the two builds in sync, which needs extra
+care when moving across branches.
```

Comment:
This looks great, thanks! FWIW I generally avoid the need to maintain my Xcode build by adding a scheme for debugging that points to my Ninja build – but that's probably getting a bit too complex for a getting started guide.

Really I'd like to clean up and land my hacky local script for generating an Xcode project on top of a Ninja build so these issues are easier to deal with.

Reply:
@hamishknight I've thought about doing that same thing (I've done something like that in the past for other project). Let me know if I can help at all